### PR TITLE
Adding variables for the vehicle api timeout for web app

### PR DESCRIPTION
**Pull request by Thomas-Franklin**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -353,6 +353,16 @@ variable "vehicle_api_client_timeout_notifier" {
   description = "Timeout in seconds of the call to the vehicle api within the notifier"
 }
 
+variable "vehicle_api_client_timeout_read_web_app" {
+  type        = "string"
+  description = "Read timeout in seconds of the call to the vehicle api within the web app"
+}
+
+variable "vehicle_api_client_timeout_connection_web_app" {
```
Comment (aaronlong) [inline variables.tf]: Should these variables match the naming of MOT_TEST_REMINDER? :)

Comment (Thomas-Franklin) [inline variables.tf]: The ones that are seen by the lambda will match the naming :) These are just mapped to the MOTR repo, and match the timeout naming of the notifier - but for consistency I am happy to update.

File: lambda.tf [hunk 2]
```diff
@@ -15,20 +15,22 @@ resource "aws_lambda_function" "MotrWebHandler" {
...
+      MOT_TEST_REMINDER_INFO_API_URI                       = "${var.mot_test_reminder_info_api_uri == "" ? "https://${aws_api_gateway_rest_api.MotrWeb.id}.execute-api.${var.aws_region}.amazonaws.com/${var.environment}/mot-test-reminder-mock/{registration}" : var.mot_test_reminder_info_api_uri}"
+      GOV_NOTIFY_API_TOKEN                                 = "${var.gov_notify_api_token}"
+      CONFIRM_EMAIL_NOTIFICATION_TEMPLATE_ID               = "${var.confirm_email_notification_template_id}"
+      CONFIRMATION_TEMPLATE_ID                             = "${var.confirmation_template_id}"
+      BASE_URL                                             = "${var.base_url == "" ? "https://${aws_api_gateway_rest_api.MotrWeb.id}.execute-api.${var.aws_region}.amazonaws.com/${var.environment}/" : var.base_url}"
+      WARM_UP                                              = "${var.webapp_warm_up}"
+      MOT_TEST_REMINDER_INFO_API_CLIENT_READ_TIMEOUT       = "${var.vehicle_api_client_timeout_read_web_app}"
+      MOT_TEST_REMINDER_INFO_API_CLIENT_CONNECTION_TIMEOUT = "${var.vehicle_api_client_timeout_connection_web_app}"
```
Comment (sebolabs) [inline lambda.tf]: why do we have different variables names on both sides: Lambda Env vars vs Terraform vars?

File: variables.tf (hunk 1 above)
Comment (Thomas-Franklin) [inline variables.tf]: I've updated it now anyway :)

**Pull request merged**